### PR TITLE
Container Card Contact

### PR DESCRIPTION
**Pull request by mattsskea**
Added Container Contact Card and icons

### CODE REVIEW

File: components/container-cards/contact-card/contact-card.hbs [hunk 1]
```diff
@@ -0,0 +1,216 @@
+<div class="container">
+  <div class="row">
+    <div class="col-12">
+      <h1>Sizes & variations</h1>
+      <h2 style="font-size: 14px;	font-weight: bold; color: #6D6F71;">WITH IMAGE</h2>
+    </div>
+
+    <div class="col-6">
+      <h4 style="font-size: 14px;	font-weight: bold; color: #6D6F71;">Large</h4>
```
Comment (mmussmann) [inline components/container-cards/contact-card/contact-card.hbs]: you could just use the typography classes instead.

File: components/container-cards/contact-card/contact-card.scss [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+      background: $gray-3;
+      margin: 0px;
+      padding: 20px 20px 32px 20px;
+      height: 72px;
+    }
+
+    .msds-container-contact-card__body-contact-person {
+      margin-top: 18px;
+      padding: 0px 20px;
+      // margin-top: 24px;
```
Comment (mmussmann) [inline components/container-cards/contact-card/contact-card.scss]: cleanup commented code

File: components/container-cards/contact-card/contact-card.scss [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+  &__body-contact-position {
+    @include msds-font('body-1');
+    color: $gray-8;
+    margin-bottom: 24px;
+  }
+
+  &__body-contact-email {
+    color: $clear-blue;
+    font-size: 16px;
```
Comment (mmussmann) [inline components/container-cards/contact-card/contact-card.scss]: should be something in typography that you can use

File: components/container-cards/contact-card/contact-card.scss [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+    .msds-container-contact-card__link {
+      color: $clear-blue;
+      text-decoration: none;
+    }
+  }
+
+  &__body-contact-social-media {
+    font-size: 16px;
+    line-height: 22px;
+    .msds-container-contact-card__icon {
```
Comment (mmussmann) [inline components/container-cards/contact-card/contact-card.scss]: add a linebreak, makes it a bit easier to read

File: components/container-cards/contact-card/contact-card.hbs (hunk 1 above)
Comment (mattsskea) [reply components/container-cards/contact-card/contact-card.hbs]: I have done the corrections based on your feedback, now using:
 <div class="msds-text-display-3">Sizes & variations</div>
 <div class="msds-text-body-1">WITH IMAGE</div>

File: components/container-cards/contact-card/contact-card.scss (hunk 3 above)
Comment (mmussmann) [reply components/container-cards/contact-card/contact-card.scss]: What about this one?

**Pull request merged**